Sources/Plasma/FeatureLib/pfMetalPipeline/plMetalDevice.cpp

```diff
@@ -131,6 +133,15 @@ bool plMetalDevice::InitDevice()
     // Only known tiler on Apple devices are Apple GPUs.
     // Apple recommends a family check for tile memory support.
     fSupportsTileMemory = fMetalDevice->supportsFamily(MTL::GPUFamilyApple1);
+    
+    // Only known tiler on Apple devices are Apple GPUs.
+    // Apple recommends a family check for tile memory support.
+    fSupportsTileMemory = fMetalDevice->supportsFamily(MTL::GPUFamilyApple1);
+    fSupportsDXTTextures = fMetalDevice->supportsBCTextureCompression();
+    
+    if (!fSupportsDXTTextures) {
+        hsDebugPrintToTerminal(ST::format("Render device \"%s\" does not support DXT textures. Falling back on software decompression. Performance will be slower.", fMetalDevice->name()->cString(NS::StringEncoding::UTF8StringEncoding)));
```

Comment:
Please use `hsStatusMessage` or similar instead, unless there's a reason why this message should never go into a log file. One reason why I renamed `DebugMsg` to this long name is to make it clear that it's *not* a general-purpose debug logging function :)

```suggestion
        hsStatusMessageF("Render device \"{}\" does not support DXT textures. Falling back on software decompression. Performance will be slower.", fMetalDevice->name()->cString(NS::StringEncoding::UTF8StringEncoding));
```

I'm not quite clear on the rebase state of this PR - if you don't have the `ST::string` changes for `hsStatusMessage` yet, write it like this instead:

```suggestion
        hsStatusMessage(ST::format("Render device \"{}\" does not support DXT textures. Falling back on software decompression. Performance will be slower.", fMetalDevice->name()->cString(NS::StringEncoding::UTF8StringEncoding)).c_str());
```